assemble/bin/accumulo

```diff
@@ -112,7 +112,7 @@ shell)   export ACCUMULO_OPTS="${ACCUMULO_GENERAL_OPTS} ${ACCUMULO_SHELL_OPTS}"
 esac
 
 XML_FILES="${ACCUMULO_CONF_DIR}"
-LOG4J_JAR=$(find -H "${HADOOP_PREFIX}/lib" "${HADOOP_PREFIX}"/share/hadoop/common/lib -name 'log4j*.jar' -print 2>/dev/null | head -1)
+LOG4J_JAR=$(find -H "${HADOOP_PREFIX}/lib" "${HADOOP_PREFIX}"/share/hadoop/common/lib -name 'log4j*.jar' -not -name 'log4j-core*.jar' -print 2>/dev/null | head -1)
```

Comment:
Are you OK with `-regex '.*/log4j-[0-9].*.jar'` being used instead of `-name 'log4j*.jar' -not -name 'log4j-core*.jar'`? The regex will prevent log4j-core and is a little more exact.